packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import KeyCode from 'keycode-js';
+import DropdownButtonBase from './_DropdownButtonBase';
+import styles from './DropdownButton.module.scss';
+import Button from './_Button';
+import SplitButton, { Variants as SplitButtonVariants } from './SplitButton';
+
+const cx = classNames.bind(styles);
+
+const Variants = {
+  NEUTRAL: 'neutral',
+  EMPHASIS: 'emphasis',
+  GHOST: 'ghost',
+};
+
+const propTypes = {
+  /**
+   * The options to display in the dropdown. Should be of type `Button`.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Determines whether the dropdown should be disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Determines whether the component should have block styles applied. The dropdown will match the component's width.
+   */
+  isBlock: PropTypes.bool,
+  /**
+   * Sets the text that will be shown on the dropdown button.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Sets the styles of the component.
+   * Must be one of 'neutral', 'emphasis', or 'ghost'.
```

Comment:
Can we add `neutral` is default?

Reply:
Do we need to add this twice? Does it already display within the generated props table?